sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -531,4 +534,7 @@ async def write(self, s):
         See http://tools.ietf.org/html/rfc5234
         http://tools.ietf.org/html/rfc6455#section-5.2
         """
-        await self.ws.send_bytes(s)
+        try:
+            await self.ws.send_bytes(s)
+        except asyncio.TimeoutError as te:
+            raise ConnectionError('send timed out (%s)' % te)
```

Comment:
just checking, but is this the only except that you'd have to catch + re-raise?

Reply:
Probably a few more, would need to dig in the library to see what else is possible but most are asyncio.TimeoutError types

Reply:
Libba will be adding those in her PR